src/content/docs/apis/lookups-service-api/lookups-service-api.mdx

```diff
@@ -10,31 +10,36 @@ redirects:
   - /docs/apis/lookup-service-api/lookup-service-api
 ---
 
-Use the NRQL Lookups API to create and manage [lookup tables](/docs/logs/ui-data/lookup-tables-ui/#overview).
+If you have custom data that's not reported to New Relic out of the box, you likely use [lookup tables](/docs/logs/ui-data/lookup-tables-ui/#overview) to query certain kinds of telemetry data. With our NRQL lookups API, you can programmatically create, update, or delete your lookup tables. This is a good option for users who want a more efficient way to query and handle their custom data.  
 
-## Before you start
-
-The NRQL Lookups API is a REST API that allows you to manage lookup tables programmatically. As another option you can also [manage lookup tables through our UI](/docs/logs/ui-data/lookup-tables-ui/#lookup-ui).
+The NRQL lookups API is a REST API that allows you to manage lookup tables programmatically. If you don't want to work with REST APIs, you can [manage lookup tables via our UI](/docs/logs/ui-data/lookup-tables-ui/#lookup-ui).
 
 ## HTTP endpoints [#http-endpoints]
 
-### Base URL
+### Set the base URL for your New Relic account
 
-Use the base URL that's applicable for your New Relic account in your API calls.
+The base URL is the initial part of the API URL, defined by which [data center](/docs/accounts/accounts-billing/account-setup/choose-your-data-center) the API needs to access.
 
-United States (US) endpoint:
+The United States (US) endpoint is:
 
 ```
 https://nrql-lookup.service.newrelic.com
 ```
 
-[European Union](/docs/using-new-relic/welcome-new-relic/get-started/introduction-eu-region-data-center) (EU) endpoint:
+The European Union (EU) endpoint is:
 
 ```
 https://nrql-lookup.service.eu.newrelic.com
 ```
 
-### Endpoints
+### Choose an endpoint
+
+    Add an endpoint to the base URL depending on the action you want to take. There are two variables you'll need to update because they're specific to your account:
```

Comment:
In the context of this sentence, "choose" actually makes more sense.  But endpoint is still a bit confusing. 
I think this should say "Choose an HTTP method and path based on the action you want to take. The path will be appended to the the base URL." And make 'base URL' link to the base URL section above.

In the table that follows, I'd create 3 columns:
* Action  - same as the current first column (the current "Method" column)
* HTTP Method - The first part of the current "Endpoint" column (POST, PUT, GET, DELETE)
* Path - The second part word of the current "Endpoint" column
* Description - Same as the current "Description"